catalog/pages/banner/index.js

```diff
@@ -26,7 +26,7 @@ class FunctionalBannerDemo extends Component {
           href="https://www.ticketmaster.com/"
           linkProps={{ target: "_blank" }}
           isOpen={isOpen}
-          variant="info"
+          variant="custom"
           onRequestClose={this.onRequestClose}
```

Comment:
Could you please leave this Demo component as-is? You can create additional one to test the "custom" variant